### PR TITLE
How to create a erc1155 contract and connect a Celo wallet in your app

### PR DESCRIPTION
**Pull request by lucasespinosa28**
tutorial for Mobile DeFi: Prize For Best Celo Learn Tutorial!

I made this tutorial for the hackathon  ,may need some review

### CODE REVIEW

**Comment by lucasespinosa28** [comment]
any feedback?

**Review by yash-sharma1 — CHANGES_REQUESTED** [review]
@lucasespinosa28 appreciate your work :) 
Just a few points to be noted and change - 

1. Please try to explain the code and include tips & do's/don'ts if there are any for users which they need to keep in mind.
2. Add the conclusion and what's next at the end of the tutorial.
3. Provide expected output after necessary steps so users can get an idea if they run into any issues.
4. If the tutorial is too long feel free to split it into two parts accordingly, if needed

**Comment by lucasespinosa28** [comment]
I remade the tutorial using the feedback

I didn't want to miss the hackathon, but I  will need more advice and I will have to fix some spelling errors

**Review by pyr0gan — COMMENTED** [review]
There are some grammar and punctuation issues that could be fixed, but we're not editing copy at the New York Times here 😉 
I've left a couple of more specific comments, it would be great if you could tidy those up. It's difficult to be critical of grammar when I completely understand that not everyone is going to write University thesis level documentation - however it is important to maintain readability throughout our articles on Learn. I am happy to give a comprehensive review of all the language in this article and how it might be fixed, but I am not trying to make you feel self conscious about your writing 😄

**Comment by pyr0gan** [comment]
> @lucasespinosa28 appreciate your work :)
> Just a few points to be noted and change -
> 
> 1. Please try to explain the code and include tips & do's/don'ts if there are any for users which they need to keep in mind.
> 2. Add the conclusion and what's next at the end of the tutorial.
> 3. Provide expected output after necessary steps so users can get an idea if they run into any issues.
> 4. If the tutorial is too long feel free to split it into two parts accordingly, if needed

@lucasespinosa28 When @yash-sharma1 marks the changes as complete we can look at merging. I have some other suggestions, you can take those into account also 😄

File: network-documentation/celo/tutorial/celo-hardhat-deploy-and-nft-app.md [hunk 1]
```diff
@@ -266,7 +171,9 @@ describe("TinyVillage Test", function() {
         const balance = await tinyVillage.balanceOf(accounts[0].address,0)
         expect(1).to.equal(Number(balance.toString()));
     });
-
+```
+The more complex test, to mint a castle we'll need to mint every other NFT,For this tutorial don't take a long time, we won't test all the minting functions
```
[the remark this answers is not in the document]
Comment (pyr0gan) [reply network-documentation/celo/tutorial/celo-hardhat-deploy-and-nft-app.md:175]: @lucasespinosa28 We need an 'about the author' section with social media links, and a Conclusion section would also be good to have. Just to wrap up the tutorial 😄 Please submit by end of day 04/23 so we can do a final review.

**Review by lucasespinosa28 — COMMENTED** [review]
adding suggestions

**Comment by lucasespinosa28** [comment]
i added about and conclusion,i did a lot of small changes 

I think the testing part could get better, but I had no idea how to explain it better without getting too long

**Review by yash-sharma1 — APPROVED** [review]

**Pull request merged**